mgmtfn/dockplugin/netDriver.go

```diff
@@ -409,6 +476,42 @@ func programExternalConnectivity(w http.ResponseWriter, r *http.Request) {
 func revokeExternalConnectivity(w http.ResponseWriter, r *http.Request) {
 	resp := api.RevokeExternalConnectivityResponse{}
 
+	logEvent("revoke externalConnectivity")
+
+	content, err := json.Marshal(resp)
+	if err != nil {
+		httpError(w, "Could not generate a response", err)
+		return
+	}
+
+	w.Write(content)
+}
+
+func discoverNew(w http.ResponseWriter, r *http.Request) {
+	resp := api.DiscoveryResponse{}
+
+	logEvent("discoverNew")
+
+	content, _ := ioutil.ReadAll(r.Body)
```

Comment:
Please always check errors when they are returned from a function.  We've had users hitting panics in the past because of functions returning errors which were not checked